### ISSUE TITLE
Enum ranges

### ISSUE THREAD

**Issue by chris-martin** [typeclasses/haskell-phrasebook#7]
A few ideas:

```haskell
import Data.Time.Calendar

main =
  do
    putStrLn (show [3 .. 8])
    putStrLn (show ['a' .. 'z'])
    putStrLn (show [Tuesday .. Friday])
```

```
$ runhaskell range.hs
[3,4,5,6,7,8]
"abcdefghijklmnopqrstuvwxyz"
[Tuesday,Wednesday,Thursday,Friday]
```

Might be nice to also show:

- How to define your own type that derives `Enum`
- Leaving off one of the bounds: `[.. x]` and `[x ..]`
- Using `[minBound .. maxBound]` to list all values

**Comment by chris-martin** [comment]
PR: #17

**Comment by argumatronic** [comment]
Closed due to merging of PR #17.
